components/Richtext.js

```diff
@@ -29,6 +32,8 @@ export const Richtext = ({ text }) => {
   }
 
   const collapsibleRenderer = ({ TDefaultRenderer, tnode,  ...props }) => {
+    const isCollapsible = tnode.domNode.attribs.class === 'collapsible-item'
+
     if (isCollapsible(tnode)){
```

Comment:
this should now just be `isCollapsible` instead of a function

Reply:
aiai...okay fixed